### PR TITLE
Integration with bigbluebutton

### PR DESCRIPTION
**Pull request by cui-soft**
How are you, everybody?

I integrated [bigbluebutton](https://bigbluebutton.org/) on gibbon project.
So teacher and students can use video chat on planners.

### CODE REVIEW

**Comment by fvlasie** [comment]
This is the real pull request for BigBlueButton with updated code so I will close the other one. Thank you !

File: modules/Staff/src/Messages/VideoChatRequest.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+    public function getTitle() : string
+    {
+        return __('Video Chat Request');
+    }
+
+    public function getText() : string
+    {
+        return __("You have a meeting with ".$absence['from'].".");
+    }
```
Comment (yookoala) [inline modules/Staff/src/Messages/VideoChatRequest.php:47]: This use of the translation function doesn't work.

Suggested to change to:
```php
    public function getText() : string
    {
        return __('You have a meeting with {person}.', [
            'person' => $this->absence['from'],
        ]);
    }
```

**Review by yookoala — CHANGES_REQUESTED** [review]

File: modules/Staff/src/Messages/VideoChatRequest.php (hunk 1 above)
Comment (cui-soft) [reply modules/Staff/src/Messages/VideoChatRequest.php:47]: Ok, thank you for your suggestion.
Let me fix it.

File: modules/Staff/src/Messages/VideoChatRequest.php [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <?php
 /*
 Gibbon, Flexible & Open School System
-Copyright (C) 2010, Ross Parker
+Copyright (C) 2022, Cui wenlan
```
Comment (yookoala) [inline modules/Staff/src/Messages/VideoChatRequest.php:4]: Please maintain the original copy right owner.

Comment (cui-soft) [reply modules/Staff/src/Messages/VideoChatRequest.php:4]: OK.
I am also implementing Rocketchat integration now.
Should I insert the original copyright owner for new added files?

Comment (yookoala) [reply modules/Staff/src/Messages/VideoChatRequest.php]: 1. Please keep separated features in separated pull request. It's better for reviewers that way.
2. Please insert the original copyright owner on new files.

Comment (cui-soft) [reply modules/Staff/src/Messages/VideoChatRequest.php]: OK, I will follow you.
I've created v25.0.01 branch.
Let me delete current pull request and recreate it.

**Pull request closed without merging**